xdsl/frontend/onnx/ir_builder.py

```diff
@@ -52,3 +53,43 @@ def visit_value_info(i: ValueInfoProto, ctx: OnnxXdslMapping) -> Attribute:
     t = get_type(i.type)
     ctx.type_by_name[name] = t
     return t
+
+
+def build_module(graph: GraphProto) -> ModuleOp:
+    """Create the ModuleOp based on the onnx graph provided."""
+    module = ModuleOp([])
+
+    ctx = OnnxXdslMapping()
+    fn = visit_graph(graph, ctx)
+    module.regions[0].block.add_op(fn)
```

Comment:
```suggestion
    ctx = OnnxXdslMapping()
    fn = visit_graph(graph, ctx)
    module = ModuleOp([fn])
```
Seems like we can simplify further now that the inner function returns a func op

Reply:
Done